clients/rospy/src/rospy/impl/tcpros_base.py

```diff
@@ -473,6 +473,7 @@ def get_transport_info(self):
         Similar to getTransportInfo() in 'libros/transport/transport_tcp.cpp'
         e.g. TCPROS connection on port 41374 to [127.0.0.1:40623 on socket 6]
         """
+	# Changing this line while break tools/rosnode/src/rosnode/__init__.py at line 67
```

Comment:
Please do not refer to a line number since that is going to change if the file is being edited in the future. Instead refer to the file and a symbol, e.g. `Pattern matching this output in tools/rosnode/src/rosnode/__init__.py CONNECTION_PATTERN`.

Also this line currently uses a tab which can't be mixed with space in Python 3. Please change it to use the spaces (as the rest of the file).